### PR TITLE
[WPF] Fixes ControlTemplate sizing issue

### LINKED ISSUE #2642: ControlTemplate sizing issue in WPF
### Description
A ContentPage's content isn't resizing correctly in WPF when a ControlTemplate is used. It works fine in UWP.

### Steps to Reproduce

1. Create a Xamarin application and add a WPF project implementation.
2. Set the MainPage xaml to the following:
```xml
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="MyApp.MainPage">
    <ContentPage.ControlTemplate>
        <ControlTemplate>
            <ContentPresenter />
        </ControlTemplate>
    </ContentPage.ControlTemplate>
    
    <ContentPage.Content>
        <Grid>
            <Grid.RowDefinitions>
                <RowDefinition Height="Auto" />
                <RowDefinition Height="*" />
            </Grid.RowDefinitions>

            <Label Grid.Row="0" Text="Header" LineBreakMode="WordWrap" FontSize="24" />

            <Label Grid.Row="1"
                   Text="Lorem ipsum dolor sit amet, sed at etiam graecis. Amet dicta utroque in ius, error vituperatoribus vel ex. Cu duo veri aperiam honestatis. Quo sint movet ullamcorper cu, vero vidisse argumentum ne nec, in munere eirmod eum. Persius similique reformidans ex mei, cu quo quot nihil mediocrem."
                   LineBreakMode="WordWrap" />
        </Grid>
    </ContentPage.Content>
</ContentPage>
```

3. Resize the window.

### Expected Behavior
The Label text should wrap.

### Actual Behavior
The Label text doesn't wrap.

### Basic Information

- Version with issue: 3.0.0.446417
- Nuget Packages: Xamarin.Forms 3.0.0.446417, Xamarin.Forms.Platform.WPF 3.0.0.446417
- Affected Devices: Windows Desktop

### Screenshots
**Working** screenshot of correctly wrapping text in UWP:
image

**Not working** screenshot of no wrapping in WPF:
image

### PR DESCRIPTION
**Pull request by paymicro**
### Description of Change ###

Sending the layout of the real `Content` when resizing the `ContentPage`, if you are using a `ControlTemplate`

### Bugs Fixed ###

Fixes #2642 

### API Changes ###

/

### Behavioral Changes ###

The `Content` of the page with the template resizes when the window is resized

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Platform.WPF/FormsContentLoader.cs [hunk 1]
```diff
@@ -54,11 +54,17 @@ private object CreateOrResizeContent(FrameworkElement parent, VisualElement visu
 			//if (Debugger.IsAttached)
 			//	Console.WriteLine("Page type : " + visualElement.GetType() + " (" + (visualElement as Page).Title + ") -- Parent type : " + visualElement.Parent.GetType() + " -- " + parent.ActualHeight + "H*" + parent.ActualWidth + "W");
 
-			visualElement.Layout(new Rectangle(0, 0, parent.ActualWidth, parent.ActualHeight));
+			var actualRect = new Rectangle(0, 0, parent.ActualWidth, parent.ActualHeight);
+			visualElement.Layout(actualRect);
+
+			// ControlTemplate adds an additional layer through which to send sizing changes.
+			var contentPage = visualElement as ContentPage;
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Platform.WPF/FormsContentLoader.cs]: shouldn't you cast to `IControlTemplated` ? or are you 100% sure that this only needs to be done on ContentPages ?

Comment (paymicro) [reply Xamarin.Forms.Platform.WPF/FormsContentLoader.cs]: `IControlTemplated` does not define the `Content` property.
Only on `ContentPage` has `Content` property which we need.

Comment (StephaneDelcroix) [reply Xamarin.Forms.Platform.WPF/FormsContentLoader.cs]: what about ContentView ?

**Comment by StephaneDelcroix** [comment]
@paymicro please revert the latest commit, we don't want to introduce new API. if you're fine with handling ContentPAge and ContentView, I'm fine

**Comment by StephaneDelcroix** [comment]
build

**Comment by StephaneDelcroix** [comment]
build

**Pull request merged**